superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/transformProps.ts

```diff
@@ -291,12 +292,20 @@ export default function transformProps(
   const showValueIndexesB = extractShowValueIndexes(rawSeriesB, {
     stack,
   });
+
+  const metricsLabels = metrics
+    .map(metric => getMetricLabel(metric, undefined, undefined, verboseMap))
```

Comment:
Can we move the `verboseMap` before the other two optional parameters in the function so we don't have to pass the `undefined`s here at all? I don't see many other places mapping these values, so it might be safe to just revise the params positions, but if you feel that might introduce issues, feel free to skip this comment

Reply:
Since the map function is always going to pass the index and array as second and third args, I can't move them around, but I was thinking of changing `getMetricLabel` to accept an object, but there are over 40 places in the codebase where this map function is used, and in some cases, it's passed in directly like `map(getMetricLabel)` instead of as an anonymous function. It just seemed less risky to do it this way, but I agree that it's not pretty.